### PR TITLE
Change version referencing in spec

### PR DESCRIPTION
**Pull request by nonparibus**
This PR changes the way we refer to the specification versions on the Block Protocol website.

This is a temporary change that will be effective until such time as we expose older versions of the spec via a dropdown/other selector on the live site.

### CODE REVIEW

**Review by kachkaev — DISMISSED** [review]

File: site/src/_pages/docs/3_spec/1_core-specification.mdx [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# Core Specification 0.2
+# Core Service
```
Comment (kachkaev) [inline site/src/_pages/docs/3_spec/1_core-specification.mdx:1]: Hmm actually I’m not sure if the Core is a ‘service’. We should probably keep ‘Core Specification’ or switch to something like ‘Core’ /  ‘BlockProtocol Core’.

In the latter case we should probably update URLs too, e.g.:

- `/docs/spec/core`
- `/docs/spec/graph-service`

I can do that (including new redirects) based on the decision.

Comment (CiaranMn) [reply site/src/_pages/docs/3_spec/1_core-specification.mdx:1]: Agree with not calling core a 'service' – if we want to do that we need to rework the rest of the document, which specifies how you can define 'services', which are described in a way which is inconsistent with this document.

We can do that, just needs a bit more thinking about how we then frame the different parts of the BP – we no longer have the core spec which defines rendering and comms transport, and service spec which defines comms content, but instead a collection of services, one of which has special status.

Comment (nonparibus) [inline site/src/_pages/docs/3_spec/1_core-specification.mdx:1]:
```suggestion
# Core Specification
```

Comment (nonparibus) [reply site/src/_pages/docs/3_spec/1_core-specification.mdx:1]: Per https://github.com/blockprotocol/blockprotocol/pull/518/files#r953445265

Comment (nonparibus) [reply site/src/_pages/docs/3_spec/1_core-specification.mdx]: Actioned in https://github.com/blockprotocol/blockprotocol/pull/518/commits/6b363a9d924bff39d2f1666a2b73a38d8073b2a2

File: site/src/_pages/docs/3_spec/1_core-specification.mdx [hunk 2]
```diff
@@ -1,6 +1,6 @@
-# Core Specification 0.2
+# Core Specification
```
Comment (nonparibus) [inline site/src/_pages/docs/3_spec/1_core-specification.mdx:1]:
```suggestion
# Core
```

Upon reflection I think this suggestion from Alex is cleaner. The whole thing is already in the 'Specification' section of the sidebar/site.

File: site/tests/integration/docs.test.ts [hunk 3]
```diff
@@ -105,12 +105,10 @@ test("Docs page should contain key elements and interactions should work", async
     page.locator("text=PreviousEmbedding Blocks >> a"),
   ).toHaveAttribute("href", "/docs/embedding-blocks");
 
-  await expect(
-    page.locator("text=NextCore Specification 0.2 >> a"),
-  ).toBeVisible();
+  await expect(page.locator("text=NextCore Specification >> a")).toBeVisible();
```
Comment (nonparibus) [inline site/tests/integration/docs.test.ts:108]:
```suggestion
  await expect(page.locator("text=NextCore >> a")).toBeVisible();
```

Follows https://github.com/blockprotocol/blockprotocol/pull/518/commits/b3ee31071fe47c55d5e3096bbf7937573bc15a6f

File: site/tests/integration/docs.test.ts [hunk 4]
```diff
@@ -105,12 +105,10 @@ test("Docs page should contain key elements and interactions should work", async
...
 
   await expect(
-    page.locator("text=NextCore Specification 0.2 >> a"),
+    page.locator("text=NextCore Specification >> a"),
```
Comment (nonparibus) [inline site/tests/integration/docs.test.ts:111]:
```suggestion
    page.locator("text=NextCore >> a"),
```

Follows https://github.com/blockprotocol/blockprotocol/pull/518/commits/b3ee31071fe47c55d5e3096bbf7937573bc15a6f

**Comment by nonparibus** [comment]
@kachkaev Will we need to set up any redirects if we rename the `Core` page (or even the `Graph Service` one, which previously also contained `0.2`)? Just wanted to explicitly flag/check.

**Comment by kachkaev** [comment]
The URLs haven't changed

- https://blockprotocol.org/docs/spec/core-specification  
  https://blockprotocol.org/docs/spec/graph-service-specification
  
- https://blockprotocol-ar5luchpv.stage.hash.ai/docs/spec/core-specification  
  https://blockprotocol-ar5luchpv.stage.hash.ai/docs/spec/graph-service-specification
  
If we want to drop ‘specification’ from URLs, we’ll need redirects – yes.

**Comment by nonparibus** [comment]
@kachkaev Would you mind adding these in for me on this PR? Thanks!

File: site/tests/integration/docs.test.ts (hunk 3 above)
Comment (nonparibus) [reply site/tests/integration/docs.test.ts]: @kachkaev Can you confirm that this and https://github.com/blockprotocol/blockprotocol/pull/518#discussion_r956164262 are correct? (Following https://github.com/blockprotocol/blockprotocol/pull/518#discussion_r956162707) Thanks!

**Review by kachkaev — APPROVED** [review]

**Comment by kachkaev** [comment]
@nonparibus is away, so merging. Integration tests are failing, but this is related to #526

**Pull request merged**